packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -40,13 +56,33 @@ export const CloudPulseRegionSelect = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [regions]);
 
+    // validate launchDrakly region_ids with the ids from the fetched 'all-regions'
+    const supportedRegions = React.useMemo<Region[] | undefined>(() => {
+      const serviceTypeFlag = flags.aclpServiceTypeMap?.find(
+        (item) => item.serviceType === selectedDashboard?.service_type
+      );
+
+      const supportedRegionsIdList = serviceTypeFlag?.supportedRegionIds
+        ?.split(',')
+        .map((regionId) => regionId.trim());
+
+      if (!supportedRegionsIdList?.length) {
+        return regions;
+      }
+
+      return regions?.filter((region) =>
+        supportedRegionsIdList?.includes(region.id)
+      );
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are we disabling the eslint rules here? wouldnt it suppresses linting errors?